src/bodywork/workflow_execution.py

```diff
@@ -221,6 +222,12 @@ def _setup_namespace(config) -> str:
             k8s.create_namespace(namespace)
         else:
             _log.info(f"Using k8s namespace = {namespace}")
+            deployments = k8s.list_service_stage_deployments(namespace)
+            for name, deployment in deployments.items():
+                if deployment["git_url"] != repo_url:
+                    raise BodyworkNamespaceError(
+                        f"A project with the same name (or namespace): {namespace}, originating from a different git"   # noqa
```

Comment:
Given this string is already split over multiple lines, lets keep it within canonical the line length limit. It's rendering horribly on GitHub at this ver moment.